artemis-server/src/main/java/org/apache/activemq/artemis/core/remoting/impl/netty/NettyAcceptor.java

```diff
@@ -863,15 +863,16 @@ public NettyAcceptor setShutdownTimeout(int shutdownTimeout) {
    }
 
    private static String getProtocols(Map<String, ProtocolManager> protocolManager) {
+      if (protocolManager == null || protocolManager.isEmpty())
```

Comment:
If we are aiding readability here, what about making the variable name "protocolManagers" since it is a map of them, not a single instance.

If statements are nicer with braces I find, less screwups later.

I have no idea where this is called from but I'm also initially surprised it doesnt just throw if the map is null/empty. When is it expected/valid to have no protocols?